src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -10113,6 +10113,109 @@ class C { }
 namespace NS;
 
     class C { }
+");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        [WorkItem(56498, "https://github.com/dotnet/roslyn/issues/56498")]
+        public async Task NewInImplicitObjectCreation()
+        {
+            await AssertFormatAsync(
+                expected: @"
+class C
+{
+    void M()
+    {
+        string v = new();
+    }
+}
+",
+                code: @"
+class C
+{
+    void M() {
+        string  v     =    new   ();
+    }
+}
+");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        [WorkItem(56498, "https://github.com/dotnet/roslyn/issues/56498")]
+        public async Task NewInConstructorConstraint()
```

Comment:
Duplicate of `NewConstraint`

Reply:
i'm ok with dupes.  for all we know, something about this will trigger some special case in the future.  or, it acts as more of an oracle if we ever rewrite formatting to be sane.